platform/platform-scheduler/src/main/java/ddf/platform/scheduler/CommandJob.java

```diff
@@ -130,16 +138,14 @@ public void doExecute(JobExecutionContext context) throws JobExecutionException
             }
         } catch (UnsupportedEncodingException e) {
             LOGGER.info("Unable to produce output", e);
+        } catch (IOException e) {
+            LOGGER.info(
```

Comment:
:question: If this IOException happened, wouldn't it happen on the implicit call to close() in your try-with-resources? It doesn't sound like that from your log message.

Reply:
I updated the message to `"Error with the emptyInputStream used as a work-around for a Karaf bug. This should never happen. There is an issue with the Platform Command Scheduler implementation."`